vlsi/Makefile

```diff
@@ -99,6 +99,8 @@ SIM_TIMING_CONF = $(OBJ_DIR)/sim-timing-inputs.yml
 include $(vlsi_dir)/sim.mk
 $(SIM_CONF): $(VLSI_RTL) $(HARNESS_FILE) $(HARNESS_SMEMS_FILE) $(sim_common_files) $(dramsim_lib)
 	mkdir -p $(dir $@)
+	mkdir -p $(OBJ_DIR)/$(HAMMER_SIM_RUN_DIR)/$(notdir $(BINARY))
```

Comment:
I like this; I think we should propagate this to `sims/vcs` and `sims/verilator`. Currently the outputs for `run-binary` go into the run directory

Reply:
I think we should lump that into the `sims/vlsi` refactor (next release)